code/datums/components/item_slowdown.dm

```diff
@@ -0,0 +1,33 @@
+/// This component is used to make a mob slower when the item the component is attached to is picked up.
+/datum/component/item_slowdown
```

Comment:
Item have a `var/slowdown` that slows down the mob when held, why not use that?

Reply:
That doesn't work for some reason, I think I have only seen that variable work on clothes

Reply:
You need to set item flag `SLOWS_WHILE_IN_HAND` and call `update_equipment_speed_mods`

Reply:
Oh yeah that works, thanks